### ISSUE TITLE
Groupquests

### ISSUE THREAD

**Issue by Classic-WoW** [Helios-Empire/Helios-Tracker#157]
Hello,

We've encountered a few problems yesterday with http://db.helios-empire.eu/?quest=9946 and with http://db.helios-empire.eu/?quest=9853. Me and another player had the quest in the log. We both had to do it 2 times. Apperantly the Item that needed to be looted from the Mobs does only drop one single time. So you have to do it for each partymember that needs the item.

Its similar to the ROB Questchain - though this time it was not shared since we both had the quest already.

**Comment by Inyu** [comment]
This is happening to all qs where you have to loot an item.
In theory it should be like this for normal questitems (e.g. collect 30 naga claws ) but not for questitems dropped by elite mobs. some more examples where this occurs too
http://db.helios-empire.eu/?quest=9851
http://db.helios-empire.eu/?quest=9852
http://db.helios-empire.eu/?quest=9856
http://db.helios-empire.eu/?quest=9859

one more thing we had a case where http://db.helios-empire.eu/?item=31345 dropped only once per kill in august and we tried to get it again ~4weeks ago and then all ppl could loot it.
dont know if theres a difference between "mob drops a questgiving item" and "mobs drops a questitem"

**Comment by HEGladios** [comment]
This should be easy to fix through the database, I'll have a look.

**Comment by HEGladios** [comment]
Absolution has fixed all of the drops mentioned above and loads more.
There are probably more items that this problem applies to, please create a new issue if you come across a similar situation.
